naga/tests/snapshots.rs

```diff
@@ -316,7 +324,20 @@ impl Input {
                 ),
             },
             Err(_) => Parameters::default(),
+        };
+
+        if params.targets.is_none() {
```

Comment:
nitpick: `param.targets.unwrap_or_else(…)`?

Reply:
We're re-assigning, and I think it would be less clear this way.